docs/io/ascii/fast_ascii_io.rst

```diff
@@ -143,18 +143,12 @@ data has less than about fifteen significant figures, or if accuracy is
 relatively unimportant, this converter might be the best option in
 performance-critical scenarios.
 
-`Here
-<https://nbviewer.jupyter.org/github/astropy/astropy-notebooks/blob/master/io/ascii/conversion_profile.ipynb>`__
-is an IPython notebook analyzing the error of the fast converter, both in
-decimal values and in ULP. For values with a reasonably small number of
+For values with a reasonably small number of
```

Comment:
@taldcroft , I removed the outdated notebooks as you recommended. Please review. Thanks!